draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -194,99 +194,45 @@ Figure: SD-JWT Issuance and Presentation Flow
-It MAY further contain clear-text claims that are always disclosed to the Verifier.
-It MUST be digitally signed using the Issuer's private key.
+An SD-JWT, at its core, is a digitally signed document containing digests of the selectively discloseable claims (per claim: a random salt, the claim name and the claim value) with the clear-text claims as Disclosures outside the document.
+An SD-JWT may also contain clear-text claims that are always disclosed to the Verifier.
 
-```
-SD-JWT-DOC = (METADATA, SD-CLAIMS, NON-SD-CLAIMS)
-SD-JWT = SD-JWT-DOC | SIG(SD-JWT-DOC, ISSUER-PRIV-KEY)
-```
-
-`SD-CLAIMS` is an array of digest values that ensure the integrity of and map to the respective Disclosures.  Digest values are calculated over the Disclosures, each of which contains the claim name (`CLAIM-NAME`), the claim value (`CLAIM-VALUE`), and a random salt (`SALT`). Digests are calculated using a hash function:
-
-```
-SD-CLAIMS = (
-    HASH(SALT, CLAIM-NAME, CLAIM-VALUE)
-)*
-```
-
-`SD-CLAIMS` can also be nested deeper to capture more complex objects, as will be shown later.
-
-The Issuer further creates a set of Disclosures for all claims in the SD-JWT. The Disclosures are sent to the Holder together with the SD-JWT:
+Each digest value ensures the integrity of, and maps to, the respective Disclosure.  Digest values are calculated using a hash function over the Disclosures, each of which contains the claim name, the claim value, and a random salt. The Disclosures are sent to the Holder together with the SD-JWT in the Combined Format for Issuance.
 
-```
-DISCLOSURES = (
-    (SALT, CLAIM-NAME, CLAIM-VALUE)
-)*
-```
-
-The SD-JWT and the Disclosures are sent to the Holder by the Issuer:
-
-```
-COMBINED-ISSUANCE = SD-JWT | DISCLOSURES
-```
+## Disclosing to a Verifier
 
-## Creating Holder-Selected Disclosures
-
-To disclose to a Verifier a subset of the SD-JWT claim values, a Holder selects a subset of the Disclosures and sends it to the Verifier along with the SD-JWT.
-
-```
-HOLDER-SELECTED-DISCLOSURES = (
-    (SALT, CLAIM-NAME, CLAIM-VALUE)
-)*
-```
-
-```
-COMBINED-PRESENTATION = SD-JWT | HOLDER-SELECTED-DISCLOSURES
-```
+To disclose to a Verifier a subset of the SD-JWT claim values, a Holder selects a subset of the Disclosures and sends them to the Verifier along with the SD-JWT in the Combined Format for Presentation.
 
 ## Optional Holder Binding
 
-Some use-cases may require Holder Binding.
-
-Cryptographic Holder Binding is an optional feature, but when it is desired, `SD-JWT` must contain information about key material controlled by the Holder:
-
-```
-SD-JWT-DOC = (METADATA, HOLDER-PUBLIC-KEY, SD-CLAIMS, NON-SD-CLAIMS)
-```
+Cryptographic Holder Binding is an optional feature, but when required by the use-case, the SD-JWT must contain information about key material controlled by the Holder.
```

Comment:
```suggestion
Holder Binding is an optional feature, but when required by the use-case, the SD-JWT must contain information about key material controlled by the Holder.
```
I think it can be biometric Holder Binding too